picky-krb/src/constants.rs

```diff
@@ -141,6 +141,16 @@ pub mod cksum_types {
     pub const HMAC_SHA1_96_AES128: usize = 15;
     pub const HMAC_SHA1_96_AES256: usize = 16;
 }
+//= [2.2.1.2 TSCredentials](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-cssp/94a1ab00-5500-42fd-8d3d-7a84e6c2cf03) =//
```

Comment:
Just adding a blank line

```suggestion

//= [2.2.1.2 TSCredentials](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-cssp/94a1ab00-5500-42fd-8d3d-7a84e6c2cf03) =//
```